clients/common/cblas_interface.cpp

```diff
@@ -724,6 +724,70 @@ void cblas_gemm<rocblas_double_complex>(rocblas_operation transA,
                 ldc);
 }
 
+template <>
+void cblas_gemm<int8_t, int32_t>(rocblas_operation transA,
+                                 rocblas_operation transB,
+                                 rocblas_int m,
+                                 rocblas_int n,
+                                 rocblas_int k,
+                                 int32_t alpha,
+                                 int8_t* A,
+                                 rocblas_int lda,
+                                 int8_t* B,
+                                 rocblas_int ldb,
+                                 int32_t beta,
+                                 int32_t* C,
+                                 rocblas_int ldc)
+{
+    // cblas does not support int8_t input / int32_t output, however non-overflowing
+    // 32-bit integer operations can be represented accurately with double-precision
+    // floats, so convert to doubles and downcast result down to int32_t.
+    // NOTE: This will not properly account for 32-bit integer overflow, however
+    //       the result should be acceptable for testing.
+
+    int const sizeA = ((transA == rocblas_operation_none) ? k * lda : m * lda);
+    int const sizeB = ((transB == rocblas_operation_none) ? n * ldb : k * ldb);
+    int const sizeC = n * ldc;
+
+    std::unique_ptr<double[]> A_double(new double[sizeA]());
+    std::unique_ptr<double[]> B_double(new double[sizeB]());
+    std::unique_ptr<double[]> C_double(new double[sizeC]());
+
+    for(int i = 0; i < sizeA; i++)
+    {
+        A_double[i] = static_cast<double>(A[i]);
+    }
+    for(int i = 0; i < sizeB; i++)
```

Comment:
I think this is same as above, but it will work for transB = N (where k is unit-stride)  not for transB = T (where k has stride lda)